package.json

```diff
@@ -10,7 +10,9 @@
     "@babel/plugin-syntax-dynamic-import": "7.2.0",
     "@babel/preset-env": "7.5.5",
     "@babel/preset-react": "7.0.0",
-    "acorn-jsx": "https://github.com/geosolutions-it/acorn-jsx/tarball/master",
+    "@geosolutions/acorn-jsx": "4.0.1",
+    "@geosolutions/jsdoc": "3.4.4",
```

Comment:
Acorn and jsdoc a rereferenced together to build the proper jsdoc ? I tested and it seems to work, but I didn't introduced acorn so I don't know the customizations and how they are used. @mbarto ?

Reply:
@mbarto do you have some feedbacks on this ?

Reply:
no

Reply:
Fine. I tested and it works @MV88 tested too, so I think it should be ok.